events/wc_druidic_events.txt

```diff
@@ -24,12 +26,27 @@ character_event = {
 		fury_of_wolf_human_event_trigger = yes
 		
 		NOT = { has_global_flag = ongoing_fury_of_wolf_chain_flag }
-		
-		is_primary_war_defender = yes
+		OR = { 
+			any_courtier_or_vassal = { has_artifact = book_of_ur }
+			any_courtier_or_vassal = { has_artifact = scythe_of_elune }
+			NOT = { has_dlc = "Mystics" }
+		}
+		OR = { 
+			is_primary_war_defender = yes
+			any_neighbor_independent_ruler = { religion = death_god }
+		}
 	}
 	
 	weight_multiplier = {
-		days = 1
+		days = 1800
```

Comment:
That's wrong syntax.
```suggestion
		days = 1
```

Reply:
Won't that fire the event as soon as the trigger conditions are filled though? If someone is playing with artifacts disabled then the event would probably fire near the start of every bookmark.

Reply:
Nope. It's standart syntax of `weight_multiplier`. Take a look at the vanilla.